apps/examples/task_manager_sample/README.md

```diff
@@ -0,0 +1,24 @@
+# examples/task\_manager\_sample
+
+  This is the Task Manager 'qualification' suite.  It attempts to exercise a broad set of OS functionality.  Its coverage is not very extensive as of this writing, but it is used to qualify each TizenRT release.
```

Comment:
Please check this line. Task Manager cannot exercise a broad set of OS functionality.